hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/actions/GracefulRollingRestartRsAction.java

```diff
@@ -35,15 +35,23 @@
 public class GracefulRollingRestartRsAction extends RestartActionBaseAction {
   private static final Logger LOG = LoggerFactory.getLogger(GracefulRollingRestartRsAction.class);
 
-  public GracefulRollingRestartRsAction(long sleepTime) {
+  private boolean killMetaRs;
+
+  public GracefulRollingRestartRsAction(long sleepTime, boolean killMetaRs) {
     super(sleepTime);
+    this.killMetaRs = killMetaRs;
   }
 
   @Override
   public void perform() throws Exception {
     LOG.info("Performing action: Rolling restarting non-master region servers");
     List<ServerName> selectedServers = selectServers();
 
+    if(!killMetaRs){
+      ServerName metaServer = cluster.getServerHoldingMeta();
+      selectedServers.remove(metaServer);
+    }
+
```

Comment:
Could we move this to Action class, in order to avoid repeating it on different Action implementations? Just noticed we duplicate this in RollingBatchSuspendResumeRsAction.java.